### PR TITLE
Feat/canary css workflow

### PR DESCRIPTION
**Pull request by jd239**
## Summary of the changes
This updates removes the local canary CSS files in favour of the core web-component files. This change will keep canary in sync with the latest CSS changes as well as make it easier to maintain by having a single source of truth. The npm script for Canary React will now take the CSS files from canary once they have been copied across as part of the build process. 

This also prevents a mismatch of core and canary packages as the styling on canary will be the most up to date no matter what web-components version the consumer is using. 

## Related issue
N/A

## Checklist
N/A

### CODE REVIEW

File: packages/canary-web-components/src/global/variables.css
Comment (GCHQ-Developer-112) [inline packages/canary-web-components/src/global/variables.css:1]: There's at least one data table specific variable in this css file

**Review by GCHQ-Developer-112 — CHANGES_REQUESTED** [review]

**Review by GCHQ-Developer-112 — APPROVED** [review]
Happy to approve this PR assuming that the data tables variable - `--ic-table-row-selected` is moved over to the core variables.css file along with any other canary-specific styling that may be on the `data-tables-v2-new` branch

**Comment by jd239** [comment]
> Happy to approve this PR assuming that the data tables variable - `--ic-table-row-selected` is moved over to the core variables.css file along with any other canary-specific styling that may be on the `data-tables-v2-new` branch

Any other styling is component specific and is held within a separate CSS file.

**Review by ad9242 — CHANGES_REQUESTED** [review]
also happy to approve once --ic-table-row-selected is added to the variables.css in the web-components package

**Review by ad9242 — APPROVED** [review]

**Review by GCHQ-Developer-112 — APPROVED** [review]

**Pull request merged**